presto-yarn-package/src/main/resources/appConfig-test.json

```diff
@@ -17,6 +17,8 @@
     "site.global.catalog": "{'hive': ['connector.name=hive-cdh5', 'hive.metastore.uri=thrift://${NN_HOST}:9083'], 'tpch': ['connector.name=tpch'], 'jmx': ['connector.name=jmx']}",
     "site.global.jvm_args": "['-server', '-Xmx1024M', '-XX:+UseG1GC', '-XX:G1HeapRegionSize=32M', '-XX:+UseGCOverheadLimit', '-XX:+ExplicitGCInvokesConcurrent', '-XX:+HeapDumpOnOutOfMemoryError', '-XX:OnOutOfMemoryError=kill -9 %p', '-DHADOOP_USER_NAME=hdfs', '-Duser.timezone=UTC']",
 
+    "site.global.additional_node_properties": "plugin.config-dir=${AGENT_WORK_ROOT}/app/install/${dep.pkg.basename}/etc/catalog\nplugin.dir=${AGENT_WORK_ROOT}/app/install/${dep.pkg.basename}/plugin",
```

Comment:
how does the .j2 template substitute the array? I thought .j2 templates automatically substitutes only strings. As you see I am not doing anything in configure.py, the j2 templates are automatically populated. Do you know anyway I can substitute an array element in a .j2 template, with each element going to a  new line?